translations/translation_assistant.js

```diff
@@ -5,6 +5,8 @@ if (typeof hoot !== 'undefined') {
     hoot.require('etds61_osm')
     hoot.require('etds70_osm')
     hoot.require('emgcp_osm')
+
+    hoot.Settings.set({"writer.include.circular.error.tags": "false"});
```

Comment:
Doesn't hoot need error:circular tags to do conflation on the inputs?
I think we just want to drop them when writing the merged output

Reply:
We do use `error:circular` during conflation and it is a tag that Hoot adds with a default value (circular.error.default.value = 15) if the data doesn't have one. 

We could make adding/dropping these configurable by the UI via setting/unsetting the `writer.include.circular.error.tags` config variable or put a check in that drops it if it is the default value but I think we are OK just dropping it.